Added tutorial on high resolution items

BlazeDrake opened the pull request:
- Removed the troubleshooting items section on high resolution items because it was outdated
- Due to attachables being more complicated, I created a new page to replace it.

BlazeDrake committed:
Added tutorial on high resolution items

- Removed the troubleshooting items section on high resolution items because it was outdated
- Due to attachables being more complicated, I created a new page to replace it.

SmokeyStack commented on docs/items/high-resolution-items.md:
```diff
@@ -0,0 +1,154 @@
+---
+
+::: tip
+This tutorial uses attachables. If you do not know what an attachable is, read this page first: [Attachables](attachables.html)
```
Changed `attachables.html` to `/items/attachables`

BlazeDrake replied:
Ok, I will implement this change.

BlazeDrake committed:
Fixed attachables link

Updated the link to the page on attachables so that it lined up with the review
